Source/UI/NewUI/BillingForm/Factory/BillingFormFactory.swift

```diff
@@ -0,0 +1,25 @@
+import UIKit
+
+struct BillingFormFactory {
+    // in order
+    static let cellsStyleInOrder: [BillingFormCell] = [
+        .fullName(DefaultBillingFormFullNameCellStyle()),
+        .addressLine1(DefaultBillingFormAddressLine1CellStyle()),
+        .addressLine2(DefaultBillingFormAddressLine2CellStyle()),
+        .city(DefaultBillingFormCityCellStyle()),
+        .state(DefaultBillingFormStateCellStyle()),
+        .postcode(DefaultBillingFormPostcodeCellStyle()),
+        .country(DefaultBillingFormCountryCellStyle()),
+        .phoneNumber(DefaultBillingFormPhoneNumberCellStyle())
+    ]
+    
+    static func getBillingFormViewController(delegate: BillingFormViewModelDelegate) -> (BillingFormViewModelDelegate?, UIViewController?) {
```

Comment:
Optional : Code convention : should we get rid of get prefix and instead just have `billingFormViewController`